src/components/features/Detail/VisitModal.tsx

```diff
@@ -0,0 +1,82 @@
+import { FcInfo } from 'react-icons/fc';
+
+import styled from 'styled-components';
+
+import Button from '@/components/common/Button';
+import { Paragraph } from '@/components/common/typography/Paragraph';
+
+export default function VisitModal({
+  placeName,
+  setVisitModal,
+}: {
+  placeName: string;
+  setVisitModal: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
```suggestion
  onClose: () => void;
```

모달 상태를 true로 전달해서 열람 여부를 변경할 수 있게 유연성을 주기 보다는, 모달 컴포넌트에서는 모달 바깥의 오버레이 클릭하거나 취소 버튼을 눌렀을 때 모달을 닫으면 되므로 `setVisitModal`는 더 좁은 타입을 가지는게 올바른 방향으로 보여요. 그리고 컴포넌트를 소비하는 상위 컴포넌트에게 사용자 상호작용을 전파시키는 목적의 함수 props라면 `on*`으로 시작하도록 이름을 지어주는게 명시적으로 의미를 드러내기에 더 권장드립니다.